website/content/en/docs/overview/_index.md

```diff
@@ -88,6 +88,10 @@ By project type (replace `${IMAGE_VERSION}` with base image version in your proj
 [client-go]:https://github.com/kubernetes/client-go
 [client-go-compat]:https://github.com/kubernetes/client-go#compatibility-matrix
 
+## OLM version compatibility
```

Comment:
Would it be helpful if we document the specific OLM versions which we are supporting in [release notes](https://github.com/operator-framework/operator-sdk/releases/tag/v1.4.2)?

Reply:
cc: @estroz @jmrodri

Reply:
Not directly, since that could lead to drift. Wdyt about printing the versions packaged locally in flag help text?

Reply:
Yeah, makes sense. Will do that in a follow up